src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import './Calculator.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

Reply:
I disable the linter because I get an error: **Component should be written as a pure function**. The requirement is to make the component class-based, but the linter only accepts a pure function.